src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/SwarmAddressPicker.java

```diff
@@ -46,9 +45,7 @@
     /**
      * Constructor
      */
-
-
-    public SwarmAddressPicker(final ILogger iLogger) {
```

Comment:
Do not alter/remove any public signatures/constructors. This could break others using this library in use-cases specific to them such as programmatically configuring the stack.

Reply:
The passed DiscoveryNode is never used.  Surely best to clean that up with an appropriate release note?  Happy to revert if you wish....

Reply:
Please remove it